src/ActionRepeater.Win32/Synch/Utilities/HighResolutionWaiter.cs

```diff
@@ -45,6 +45,7 @@ private unsafe void WaitCore(long time)
 
         if (!SetWaitableTimer(_timerHandle, &time, 0, null, null, false))
         {
+            // TODO: call KillTimer when exiting app to avoid exception being thrown when app is closed during playback

```

Comment:
```suggestion
```
`KillTimer` is for window timers created with `SetTimer`, not for waitable timers (created using `CreateWaitableTimer`)